<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tag_create_popup</class>
 <widget class="QDialog" name="tag_create_popup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Start Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_start_address"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>End Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_end_address"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>117</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Saved Tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_saved_tags"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_create_new">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Create New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_new_tag">
       <property name="placeholderText">
        <string>NewTag Name...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_new_tag">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
